examples/logAxis.py

```diff
@@ -5,34 +5,38 @@
 import initExample ## Add path to library (just for examples; you do not need this)
 
 import numpy as np
-from pyqtgraph.Qt import QtGui, QtCore
 import pyqtgraph as pg
 
-
 app = pg.mkQApp("Log Axis Example")
 
 w = pg.GraphicsLayoutWidget(show=True)
-w.setWindowTitle('pyqtgraph example: logAxis')
-p1 = w.addPlot(0,0, title="X Semilog")
+w.resize(800,800)
+w.setWindowTitle('pyqtgraph example: Log Axis, or How to Recognise Different Types of Curves from Quite a Long Way Away')
+
+p0 = w.addPlot(0,0, title="Linear")
+p1 = w.addPlot(0,1, title="X Semilog")
 p2 = w.addPlot(1,0, title="Y Semilog")
-p3 = w.addPlot(2,0, title="XY Log")
-p1.showGrid(True, True)
-p2.showGrid(True, True)
-p3.showGrid(True, True)
+p3 = w.addPlot(1,1, title="XY Log")
 p1.setLogMode(True, False)
 p2.setLogMode(False, True)
 p3.setLogMode(True, True)
-w.show()
 
-y = np.random.normal(size=1000)
-x = np.linspace(0, 1, 1000)
-p1.plot(x, y)
-p2.plot(x, y)
-p3.plot(x, y)
+x = np.logspace(-1, 1, 1000) # 1000 points from 0.01 to 100
```

Comment:
the comment doesn't match the code

Reply:
Awww crap.  :)
Thank you for spotting that, I hope the updated version is more correct.